src/main/webapp/index.html

```diff
@@ -32,7 +32,8 @@ <h1>CodeU Starter Project</h1>
     <p>This is the CodeU starter project. Click the links above to login and visit your page.
     You can post messages on your page, and you can visit other user pages if you have
     their URL.</p>
-    <p>This is your code now! Feel free to make changes, and don't be afraid to get creative!
-    You could start by modifying this page to tell the world more about your team.</p>
+    <h2>About</h2>
```

Comment:
I'm ok with this, but I'm not sure if this is meant to be part of your servlet work -- in the future, if you have multiple changes that get stacked like this, just make them multiple branches, so that it's easier to review.

Reply:
Thanks for letting me know, will keep this in mind for the future.